doc/conf.py

```diff
@@ -74,16 +74,16 @@
 
 # General information about the project.
 project = 'astroML'
-copyright = '2012-2019, Jake Vanderplas & AstroML Developers'
+copyright = '2012-2020, Jake Vanderplas & AstroML Developers'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '0.4'
+version = '1.0'
```

Comment:
now I think about it, this should not be a hard wired value here, but should come from the code.